packages/block-library/src/comments/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 2,
 	"name": "core/comments-query-loop",
```

Comment:
Let's actually also change the ID:

```suggestion
	"name": "core/comments",
```

Reasoning: It's overall much less confusing to have the ID reflect the block name (and file and folder names), especially if there are plenty of related blocks with similar names.

As for the reasoning _why_ I think we should rename the block overall, see https://github.com/WordPress/gutenberg/pull/40506#issuecomment-1105176912.
Let's add some logic to convert from the old to the new ID here: https://github.com/WordPress/gutenberg/blob/47a535914907153c62086f0d7bea521c628eca53/packages/blocks/src/api/parser/convert-legacy-block.js#L58-L68

Reply:
Adding that it seems like the stabilized Comments Query Loop block has so far only been in on GB plugin release (v13.0.0): https://github.com/WordPress/gutenberg/commit/6f7c5b573a6023c2f2ce3eecd0d6b169ce4e964c

![Bildschirmfoto 2022-04-21 um 15 07 12](https://user-images.githubusercontent.com/96308/164464668-43a0f38e-654d-4926-8fe4-e6d306ac4d9f.png)